backend/src/app.js

```diff
@@ -44,13 +40,14 @@ app.use(async (req, res, next) => {
   }
   const token = req.cookies.token || '';
   try{
-    const result = await verifyJwt(token);
-    if(!result){
-      res.sendStatus(403);
-      console.error("ERROR: Could not verify JWT");
-      return;
-    }
-    next();
+    verifyJwt(token, (err, decoded) => {
+      if(err || !decoded){
```

Comment:
```suggestion
      if(err){
```
According to the [doc](https://www.npmjs.com/package/jsonwebtoken#jwtverifytoken-secretorpublickey-options-callback), `decoded` in never falsy if `err` is falsy, so this check is redundant.